src/wallet/qi-hdwallet.ts

```diff
@@ -239,34 +239,86 @@ export class QiHDWallet extends AbstractHDWallet {
     }
 
     /**
-     * Sends a Qi transaction.
+     * Sends a transaction using the traditional method (compatible with AbstractHDWallet).
      *
-     * @param {QiTransactionRequest} tx - The transaction to send.
-     * @returns {Promise<TransactionResponse>} The transaction response.
-     * @throws {Error} If the provider is not set or if the transaction has no inputs.
+     * @param tx The transaction request.
      */
-    public async sendTransaction(tx: QiTransactionRequest): Promise<TransactionResponse> {
+    public async sendTransaction(tx: QiTransactionRequest): Promise<TransactionResponse>;
+
+    /**
+     * Sends a transaction using payment codes and specific parameters.
+     *
+     * @param recipientPaymentCode The payment code of the recipient.
+     * @param amount The amount to send.
+     * @param originZone The origin zone of the transaction.
+     * @param destinationZone The destination zone of the transaction.
+     */
+    public async sendTransaction(
+        recipientPaymentCode: string,
+        amount: bigint,
+        originZone: Zone,
+        destinationZone: Zone,
+    ): Promise<TransactionResponse>;
+
+    /**
+     * Implementation of the sendTransaction method.
+     */
+    public async sendTransaction(...args: any[]): Promise<TransactionResponse> {
```

Comment:
Instead of overloading the `sendTransaction` method I think it would be better to change the `QiTransactionRequest` to be our desired form of a transaction request which is recipientPaymentCode, amount, originZone, destinationZone.